providers/Mora/json_types.ts

```diff
@@ -0,0 +1,52 @@
+export interface ApiArgs {
+	mountPoint: string;
+	labelId: string;
+	materialNo: string;
+}
+
+export interface PackageMeta {
+	artistName: string;
+	artistNo: number;
+	cdPartNo: string | null;
+	fullsizeimage: string;
+	title: string;
+	labelCode: string;
+	labelcompanyname: string;
+	master: string;
+	distPartNo: string;
+	startDate: string;
+	mediaFormatNo: MediaFormat;
+	mediaType: MediaType;
+	trackList: Track[];
+}
+
+export interface Track {
+	arranger?: string;
```

Comment:
```suggestion
	arranger: string | null;
```
Nitpick: Properties seem to be `null` rather than missing, the same probably also holds for composer and lyrics (but I haven't seen them empty yet, I only encountered `"Not Applicable"`). Did you see any cases where they were missing or did you only use/assume the `?` marker as a safety measure? (In most cases it wouldn't matter whether something is missing, `undefined` or `null`.)

Reply:
Just an assumption, I haven't seen a case with it missing